reana_workflow_engine_snakemake/executor.py

```diff
@@ -116,22 +120,6 @@ def run(
         # it would immediately check if the output files are present
         # and fail otherwise (3 sec timeout).
```

Comment:
*question:* @mvidalgarcia, is this comment is relevant to `job.is_norun` section or the previous part? Maybe, it can be removed.

Reply:
To the `job.is_norun` section.

Reply:
Got it. I removed the comment as it doesn't look relevant with new changes.